test/foxStakingTest.ts

```diff
@@ -6,16 +6,21 @@ import { StakingWarmup } from "../typechain-types/StakingWarmup";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import { BigNumber, Contract, Signer } from "ethers";
 import ERC20 from "@openzeppelin/contracts/build/contracts/ERC20.json";
+import { StakingCooldown } from "../typechain-types";
+import { tokePoolAbi } from "./tokePoolAbi";
 
 describe("FoxStaking", function () {
   let accounts: SignerWithAddress[];
   let FOXy: Foxy;
   let foxStaking: FoxStaking;
   let fox: Contract;
+  let tFOX: Contract;
   let stakingWarmup: StakingWarmup;
+  let stakingCooldown: StakingCooldown;
 
   const FOX_WHALE = "0xF152a54068c8eDDF5D537770985cA8c06ad78aBB";
   const FOX = "0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d";
+  const tFOXAddress = "0x808D3E6b23516967ceAE4f17a5F9038383ED5311";
```

Comment:
nit: pick a const format and stick with it.  IE FOX_WHALE vs tFOXAddress (tFOX_ADDRESS)